docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+the ordering organization. You can see the ordering node running on your machine
+as `orderer.example.com`. While the test network only uses a single node ordering
+service, a real network would have multiple ordering nodes, operated by one or
+multiple orderer organizations. The different ordering nodes would use the Raft
+consensus algorithm to come to agreement on the order of transactions across
+the network.
+
+## Creating a channel
+
+Now that we have peer and orderer nodes running on our machine, we can use the
+nodes to create a Fabric channel. Channels are a private layer of communication
+that can be used by only certain network members. Channels can be used only by
+organizations that are invited the channel, and are invisible to other members
+of the network. Each channel has separate blockchain ledger. Organizations that
+have been invited can "join" their peers to the channel to store the channel
+ledger and validate the transactions on the channel.
+
+You can use the `network.sh` script to create a channel for `org1.example.com`
+and `org2.example.com` and join their peers to the channel. Run the following
+command to create a channel called `mychannel`:
+```
+./network.sh createChannel
+```
+If the command was successful, you can see the following message printed in your
+logs:
+```
+========= Channel successfully joined ===========
+```
+
+You can create additional channels by using the same `createChannel` command and
+using the channel flag to specify a new channel name. As an example, you can use
+the following command to create a second channel named `channel2`:
+```
+./network.sh createChannel -c channel2
+```
+
+## Starting a chaincode on the channel
+
+After you have created a channel, you can start using smart contracts to
+interact with the channel ledger. Smart contracts encode the business logic
+that governs that assets on the blockchain ledger. Applications run by members
+of the network can invoke smart contracts to create assets on the ledger, as
+well as change and transfer those assets. Applications can also query smart
+contracts to query the data on the ledger.
+
+To ensure that transactions are trusted, transactions created using smart
+contracts will need to be signed by multiple organizations to be added to the
+channel ledger. Multiple signatures ensures that one organization has not
+tampered with the ledger on their peer or used business logic that was not
+agreed by the network. To sign a transaction, each organization needs to invoke
+and execute the smart contract on their peer, which then signs the output of the
+transaction. If the output is consistent and has been signed by enough
+organizations, the transaction can be committed to the ledger. The policy that
+specifies which set organizations on the channel need to execute the smart
+contract is referred to as the endorsement policy, which is set for each
+chaincode as part of the chaincode definition.
+
```

Comment:
In general, I think it's a bit backwards to have so much in this paragraph and the one below about "how chaincodes work" rather than their conceptual purpose and role in the network (given comparably little space in the first paragraph). All of this stuff about signatures and executing chaincode on their peer is important, but needs to be understood in the context of smart contracts more in general. 

Also worth noting again that we should have links to where some of this stuff is described elsewhere in more detail. I worry that these sections are sort of tweeners --- not going too deep but still trying to give people a good amount of information --- which are incredibly difficult to write. You might find these sections get a lot of scrutiny from maintainers.

Reply:
I think mentioning the endorsement policy is important, but it is possible that this section can be cut down. I am going to leave as is for the moment, see what scrutiny I get.